notebooks/stable-video-diffusion/stable-video-diffusion.ipynb

```diff
@@ -9,7 +9,7 @@
     "# Image to Video Generation with Stable Video Diffusion\n",
```

Comment:
<p>how long data collaction and quantization takes?</p><p>Could you please reduce subset for internal validation adding replacement into notebook  metadata <a href="https://github.com/openvinotoolkit/openvino_notebooks/wiki/Notebooks-Development---CI-Test-Speedup" target="_blank">https://github.com/openvinotoolkit/openvino_notebooks/wiki/Notebooks-Development---CI-Test-Speedup</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Reduced `subset_size` to 4 samples